src/client/callers/agentLockAll.ts

```diff
@@ -0,0 +1,11 @@
+import { HandlerTypes, UnaryCaller } from "@matrixai/rpc";
+import NodesClosestLocalNodesGetHandler from "../../nodes/agent/handlers/NodesClosestLocalNodesGet";
+
+type CallerTypes = HandlerTypes<NodesClosestLocalNodesGetHandler>;
+
+
+const agentLockAll = new UnaryCaller<
+  CallerTypes['input'],
+  CallerTypes['output']
+>();
+export default agentLockAll;
```

Comment:
Don't forget proper spacing, linter won't fix this.

Reply:
Doing this everywhere.